### PR TITLE
Enhance export function update tests update docs

### PR DESCRIPTION
**Pull request by A0143487X-Kevin**
Added activityExport feature and formatting

### CODE REVIEW

File: src/main/java/seedu/address/logic/exportutil/ExportUtil.java [hunk 1]
```diff
@@ -57,31 +56,125 @@ public static void exportDataToFile (String fileIdentifier, String extension, St
...
+        String fileIdentifier = name.replaceAll("[^a-zA-Z0-9]", "");;
+        String matricNumber = person.getMatricNumber().toString();
+        String phone = person.getPhone().toString();
+        String email = person.getEmail().toString();
+        String address = person.getAddress().toString();
+        String gender = person.getGender().toString();
+        String yearOfStudy = person.getYearOfStudy().toString();
+        String major = person.getMajor().toString();
+
+        /*Set<Tag> tagSet = person.getTags();
```
Comment (minernchan) [inline src/main/java/seedu/address/logic/exportutil/ExportUtil.java:69]: just want to confirm that we aren't exporting the tags assigned to a member?

**Review by minernchan — COMMENTED** [review]
i think everything looks good

**Review by melkwx — APPROVED** [review]
great job!

File: src/main/java/seedu/address/logic/exportutil/ExportUtil.java (hunk 1 above)
Comment (A0143487X-Kevin) [reply src/main/java/seedu/address/logic/exportutil/ExportUtil.java:69]: I can uh. But not my focus now hahaha. maybe later

**Pull request merged**